packages/ui/src/utils/HelperUtils.ts

```diff
@@ -9,3 +9,16 @@ export function getSpacingStyles(spacing: SpacingType | SpacingType[], index: nu
 
   return undefined
 }
+
+export function removeUrlPrefix(url: string) {
```

Comment:
I believe you are trying to get a domain name from a url here? In which case using `new URL()` and grabbing hostname on that would be more stable / correct.